### PR TITLE
Kafta tests

### PR DESCRIPTION
**Pull request by afanasievM**

### CODE REVIEW

File: transaction-service/src/test/kotlin/ua/com/foxminded/restClient/kafka/KafkaConsumerServiceTest.kt [hunk 1]
```diff
@@ -1,25 +1,63 @@
...
         val request: ProtoMessage.FindTransactionsByPersonIdAndTimeRequest = Transactions.transactionRequest
-        kafkaProducerTemplate.send("transactions",request.toByteArray()).subscribe()
+        val messageList: MutableList<ProtoMessage.FindTransactionsByPersonIdAndTimeListResponse> = mutableListOf()
+        val latch = CountDownLatch(1)
+        val expectedSize = 2
+        kafkaConsumerTemplate.receiveAutoAck()
+            .handle { obj, sink ->
+                val byteArray = obj.value() as ByteArray
+                when (String(byteArray)) {
+                    "Hello" -> {}
```
Comment (ajax-shulepa-a) [inline transaction-service/src/test/kotlin/ua/com/foxminded/restClient/kafka/KafkaConsumerServiceTest.kt]: how there can be hello?

File: transaction-service/src/test/resources/application.properties [hunk 2]
```diff
@@ -18,12 +18,12 @@ spring.redis.host=localhost
...
 spring.kafka.producer.key-serializer=org.apache.kafka.common.serialization.StringSerializer
 spring.kafka.producer.value-serializer=org.apache.kafka.common.serialization.ByteArraySerializer
 spring.kafka.consumer.group-id=transactions
 spring.kafka.consumer.key-deserializer=org.apache.kafka.common.serialization.StringDeserializer
 spring.kafka.consumer.value-deserializer=org.apache.kafka.common.serialization.ByteArrayDeserializer
 spring.kafka.consumer.auto-offset-reset=latest
-kafka.consumer.topics=transactions
-kafka.producer.topic=transactions
+kafka.consumer.topics=t1
+kafka.producer.topic=t2
```
Comment (ajax-shulepa-a) [inline transaction-service/src/test/resources/application.properties]: bad naming, consider naming like test-{actual topic name}

**Review by ajax-shulepa-a — CHANGES_REQUESTED** [review]

File: transaction-service/src/test/resources/application.properties (hunk 2 above)
Comment (afanasievM) [reply transaction-service/src/test/resources/application.properties]: ok

File: transaction-service/src/test/kotlin/ua/com/foxminded/restClient/kafka/KafkaConsumerServiceTest.kt (hunk 1 above)
Comment (afanasievM) [reply transaction-service/src/test/kotlin/ua/com/foxminded/restClient/kafka/KafkaConsumerServiceTest.kt]: OMG, I can't explain it. I thought that it was some message for test when zookeeper and brokers up. But now I changed topics and I don't see Hello message.
May be it was in some topics, when i try to test it firstly.

**Review by ajax-shulepa-a — APPROVED** [review]

**Pull request merged**